### PR TITLE
Install the UX data into a single .tar.gz file

### PR DESCRIPTION
**Pull request by hughsie**
This will provide much better compression (-20%) than gzip on them individually
and also allows us to build the capsule UX images as part of the build stage.

Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@superm1 not sure why I didn't think of this before; what do you think of this approach? There's some more simplification needed in make-images.py but I didn't want to sink time in it if you hated the idea.

**Comment by superm1** [comment]
I love the idea!

**Review by superm1 — COMMENTED** [review]
`usr/share/locale` can be removed from `contrib/debian/fwupd.install` IIRC now

File: plugins/uefi-capsule/make-images.py [hunk 1]
```diff
@@ -6,177 +6,153 @@
...
+            continue
+        yield x
+    yield "en"
+
+
+def main(args) -> int:
+
+    domain = "fwupd"
+    gettext.textdomain(domain)
+    gettext.install(domain, args.localedir)
```
Comment (superm1) [inline plugins/uefi-capsule/make-images.py:42]: do you still need this?

File: plugins/uefi-capsule/meson.build [hunk 2]
```diff
@@ -97,7 +97,22 @@ install_data(['uefi_capsule.conf'],
...
+custom_target('ux-capsule-tar-gz',
+  input : [
+    join_paths(meson.source_root(), 'po', 'LINGUAS'),
+  ],
+  output : 'ux-capsule.tar.gz',
+  command : [
+    python3.path(),
+    join_paths(meson.current_source_dir(), 'make-images.py'),
+    '--linguas', '@INPUT@',
+    '--localedir', localedir,
```
Comment (superm1) [inline plugins/uefi-capsule/meson.build:109]: do you still need this?

File: plugins/uefi-capsule/make-images.py (hunk 1 above)
Comment (hughsie) [reply plugins/uefi-capsule/make-images.py:42]: Urgh, yes, and no. If you ninja install then run the script then the .mo files exist and it works. From a clean checkout this gets run before the .mo files get created. From a 40,000ft view its insane we're creating a BMP file from a .mo file from a .po file, and I think we can skip out the middle step.

**Review by superm1 — APPROVED** [review]

**Comment by hughsie** [comment]
Slight change; I switched from gzip to xz -- it took the archive size from 2.6Mb to 1.1Mb! I also added in a few more useful screen sizes.

**Pull request merged**